web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -292,7 +292,9 @@
       </div>
 
       <Textbox
-        className="migrate__input-field"
+        className="migrate__input-field {!isAmountValid
```

Comment:
You could use our usual `makeClassName` helper for consistency, which also makes easier adding or removing class names as the component evolves.